pkg/security/security.go

```diff
@@ -0,0 +1,229 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package security
```

Comment:
What's the relation with `istio.io/istio/security`? Why cannot move it there?

Reply:
It's shared interfaces. We moved agent to istio.io/istio/pkg, we'll implement SDS in istio.io/istio/pilot/pkg/xds, etc

the istio/security package has implmentations of those interfaces.

Reply:
And in general the code in istio.io/istio/pkg is supposed to avoid depending on the actual components - it's supposed to be shared code across components.